### ISSUE TITLE
[BUG] Following instructions for podman install; "Only supported with OpenShift bundles"

### ISSUE THREAD

**Issue by ianw** [code-ready/crc#3309]
### General information

  * OS: Linux CentOS 9-stream
  * Hypervisor: KVM 
  * Did you run `crc setup` before starting it (Yes/No)?
  * Running CRC on: VM

## CRC version
```bash
CRC version: 2.7.1+a8e9854
OpenShift version: 4.11.0-0.okd-2022-07-29-154152
Podman version: 4.1.1
CRC VM:          Running
OpenShift:       
Podman:          4.1.1
Disk Usage:      1.777GB of 32.74GB (Inside the CRC VM)
Cache Usage:     2.485GB
Cache Directory: /home/zuul/.crc/cache
- consent-telemetry                     : no
- kubeadmin-password                    : password
- preset                                : podman
- pull-secret-file                      : /home/zuul/pull-secret
- skip-check-kvm-enabled                : true
- skip-check-virt-enabled               : true
```
  
### Steps to reproduce

I have followed the instructions at 

https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5/html/getting_started_guide/using_gsg#accessing-the-openshift-cluster-with-oc_gsg

Running ```crc os-env``` gives ```Only supported with OpenShift bundles``` and the ```oc``` tool is not installed.

I am clearly missing something about what the ```podman``` preset does?  I thought that this would setup the openshift containers with podman giving me a small development environment.  There's no containers running and I'm not sure what it's purpose actually is?

Is there any documentation I am missing about what the "podman" presets actually do?

I'm looking to translate some CI jobs from CentOS 7 era OpenShift 3 that used PaaS repos and a bunch of origin docker images [1].  The VM's we test in are "only" 8gb, so I can't even get crc started with the openshift backend.  Another complication is that we are testing on a VM, and some of them don't supported nested KVM.  Is it even possible to really start OpenShift 4 in this environment, or am I tilting at windmills?

[1] https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/ensure-openshift/tasks/main.yaml

**Comment by anjannath** [comment]
The `podman` preset provides a *Podman runtime* it doesn't deploy OpenShift 4 using containers on the host, the `podman` preset is mostly useful for macOS and windows to have a container runtime available

The minimum requirements for running OCP are mentioned in the docs, https://crc.dev/crc/#_for_openshift_container_platform you need at leas 9gb ram

**Comment by gbraad** [comment]
Perhaps a microshift option would be more suited for this, but this is currently on hold due to the rebase they are performing. We will be looking into this soon

However 8GB would not suffice for running OpenShift 4.x as our default is really the bare minimum without an application running.
